pkg/cli/admin/release/info.go

```diff
@@ -207,7 +211,7 @@ const defaultGraphURL = "https://api.openshift.com/api/upgrades_info/v1/graph"
 // replaceStableSemanticArgs attempts to look up known major versions in existing public stable
 // channels.
 // TODO: perfom graph lookups from the cluster's graph endpoint and channel in preference
```

Comment:
@Prashanth684 I do need some help here.. The way the current look up sequence is:

When connected to a cluster:
- `oc adm release info` will return that clusters release container and its contents
- `oc adm release info 4.x.x` will return that clusters release container ONLY if the release version matches the value passed.  If the `arch` value isn't passed it will return x86

So my question is do we want `oc adm relase info --graph-arch=<arch different than the cluster>` to reach to the graph api endpoint and pull the right payload? I think this is doable, but not seeing an easy/obvious why to get a clusters architecture w/o cracking open the release container?